blockchain/blockchain.go

```diff
@@ -634,10 +576,18 @@ func (b *Blockchain) WriteHeadersWithBodies(headers []*types.Header) error {
 		}
 	}
 
-	// Write the actual headers
-	for _, h := range headers {
+	// Write the actual headers in separate batches for now
```

Comment:
Why in separate batches?

Reply:
it would  require a lot of code changes if we write everything in one batch (because writing new header requires reading of parent headers data). Good thing is that this function is only used by tests